src/app/study-sequence/study-sequence.component.scss

```diff
@@ -14,15 +13,13 @@
 }
 
 .sequence-builder {
-    outline: 1px dashed magenta;
     grid-column-start: 3;
     grid-column-end: 4;
     display: grid;
     grid-template-rows: auto auto auto;
 }
 
 .sequence-select {
```

Comment:
Can you apply the background colors to this and the check-show-answer classes. You should create a new file called _study-sequence-theme.scss and apply theme stylings in there. This is where you do the mixin things, and don't forget to add it to the global style sheet. To differentiate the sequence select menu and the background, you can outline the "select-title" class.